jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -18,14 +18,111 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('entity-details', function() {
+YUI.add('entity-details', function(Y) {
+
+  var utils = Y.namespace('juju.views.utils');
```

Comment:
Utils is now passed into Charmbrowser, and it passes pluralize into `EntityDetails`. That may change as `EntityDetails` grows to render the rest of the details page, but it's a start.